src/config.c

```diff
@@ -2840,7 +2840,7 @@ standardConfig configs[] = {
     createIntConfig("timeout", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.maxidletime, 0, INTEGER_CONFIG, NULL, NULL), /* Default client timeout: infinite */
     createIntConfig("replica-announce-port", "slave-announce-port", MODIFIABLE_CONFIG, 0, 65535, server.slave_announce_port, 0, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("tcp-backlog", NULL, IMMUTABLE_CONFIG, 0, INT_MAX, server.tcp_backlog, 511, INTEGER_CONFIG, NULL, NULL), /* TCP listen backlog. */
-    createIntConfig("cluster-port", NULL, IMMUTABLE_CONFIG, 0, 65535, server.cluster_port, 0, INTEGER_CONFIG, NULL, NULL),    
+    createIntConfig("cluster-port", NULL, MODIFIABLE_CONFIG, 0, 65535, server.cluster_port, 0, INTEGER_CONFIG, NULL, NULL),
```

Comment:
actually I didn't think of it yet, this parameter is added by 7.0.. 
but i see other ports is MODIFIABLE_CONFIG like (I haven't tested it, feel a little risky, i can remove it):
```
createIntConfig("port", NULL, MODIFIABLE_CONFIG
createIntConfig("cluster-announce-bus-port", NULL, MODIFIABLE_CONFIG
createIntConfig("cluster-announce-port", NULL, MODIFIABLE_CONFIG
createIntConfig("cluster-announce-tls-port", NULL, MODIFIABLE_CONFIG
createIntConfig("replica-announce-port", "slave-announce-port", MODIFIABLE_CONFIG
```

Reply:
Yeah I think reverting back to immutable is the right thing to do. Otherwise this change opens up a window for inconsistency when someone accidentally changed the cluster bus port. The config-set command won't update the cluster bus port at runtime.